channels-rke2.yaml

```diff
@@ -1818,3 +1844,29 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1700
     featureVersions: *featureVersions-v1
+  - version: v1.27.12-rc1+rke2r1
+    minChannelServerVersion: v2.8.0-alpha1
```

Comment:
@briandowns when updating for the non-rc versions, we should also change this to be `v2.7.11-alpha1` so that it is present in the v2.7 release line